src/main/resources/xslt/outputs/odt/models.xsl

```diff
@@ -1,544 +1,641 @@
+		
+		<text:p text:style-name="Module"><xsl:value-of select="enoodt:get-label($source-context, $languages[1])"/></text:p>
+		
+		<!-- Go to the children -->
+		<xsl:apply-templates select="eno:child-fields($source-context)" mode="source">
+			<xsl:with-param name="driver" select="." tunnel="yes"/>
+		</xsl:apply-templates>
+	</xsl:template>
+	
+	<!-- Match on the SubModule driver: write the sub-module label -->
+	<xsl:template match="SubModule" mode="model">
+		<xsl:param name="source-context" as="item()" tunnel="yes"/>
+		<xsl:variable name="languages" select="enoodt:get-form-languages($source-context)" as="xs:string +"/>
+		
+		<text:p text:style-name="SubModule"><xsl:value-of select="enoodt:get-label($source-context, $languages[1])"/></text:p>
+		
+		<!-- Go to the children -->
+		<xsl:apply-templates select="eno:child-fields($source-context)" mode="source">
+			<xsl:with-param name="driver" select="." tunnel="yes"/>
+		</xsl:apply-templates>
+	</xsl:template>
+	
+	<xsl:template match="xf-input" mode="model">
+		<xsl:param name="source-context" as="item()" tunnel="yes"/>
+		<xsl:variable name="languages" select="enoodt:get-form-languages($source-context)" as="xs:string +"/>
+		<xsl:variable name="typeResponse" select="enoodt:get-type($source-context)"/>
+		<xsl:variable name="lengthResponse" select="enoodt:get-length($source-context)"/>
+		<xsl:variable name="numberOfDecimals" select="enoodt:get-number-of-decimals($source-context)"/>
+		<xsl:variable name="minimumResponse" select="enoodt:get-minimum($source-context)"/>
+		<xsl:variable name="maximumResponse" select="enoodt:get-maximum($source-context)"/>
+		<xsl:variable name="questionName" select="enoodt:get-question-name($source-context)"/>
+		<xsl:variable name="questionLabel" select="enoodt:get-label($source-context, $languages[1])"/>
+		
+		<xsl:if test="$questionName != ''">
+			<text:p text:style-name="QuestionName">
+				<xsl:value-of select="concat('[',$questionName,']')"/>
+			</text:p>
+		</xsl:if>
+		<xsl:if test="$typeResponse!=''">
+			<text:p text:style-name="Format">
+				<xsl:choose>
+					<xsl:when test="$typeResponse='text'">
+						<xsl:value-of select="concat('Car ',$lengthResponse)"/>
+					</xsl:when>
+					<xsl:when test="$typeResponse='number' and fn:string-length($numberOfDecimals)>0">
+						<xsl:value-of select="concat('num ',fn:substring-before($minimumResponse,'.'),'..',fn:substring-before($maximumResponse,'.'),' - ',$numberOfDecimals,' chiffre(s) après la virgule')"/>
+					</xsl:when>
+					<xsl:when test="$typeResponse='number' and fn:string-length($numberOfDecimals)=0">
+						<xsl:value-of select="concat('num ',$minimumResponse,'..',$maximumResponse)"/>
+					</xsl:when>
+					<xsl:when test="$typeResponse='date'">
+						<xsl:value-of select="$typeResponse"/>
+					</xsl:when>
+					<xsl:when test="$typeResponse='boolean'">
+						<xsl:value-of select="'Booléen'"/>

```

Comment:
Take a look at the 'conversion-table' mode implementation in v2beta/ddi2pdf, it implements a mechanism for mapping input vocabulary to ouptut ones.